client/src/routes/mardown-editor/index.js

```diff
@@ -31,9 +32,34 @@ Start using Great Note today and unlock a world of possibilities for organizing
 const MarkdownEditor = () => {
   const [markdownText, setMarkdownText] = useState(defaultMarkdown);
 
+  const handleSave = async () => {
+    const content_url = markdownText;
+
+    try {
+      let apiURL = getApiURL();
+      const response = await fetch(apiURL +'/api/notes/save-note', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ content_url }),
+      });
+
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+
+      const result = await response.json();
+      console.log('Note saved successfully:', result);
+    } catch (error) {
+      console.error('Error saving note:', error);
+    }
+  };
+
   return (
     <main id="editor-page">
-      <section className="">
+      <section className="editor-section">
+        <button className="save-button" onClick={handleSave}>Save</button>
```

Comment:
and just remember that Arinaho has added a save button so you're gonna have merge conflicts once everything goes into main. Probably our first item to look at when we get together, is getting main up to date again.

Reply:
Ah yes. ill remove it in the conflict resolver